views/layout.hbs

```diff
@@ -1,6 +1,30 @@
 <!DOCTYPE html>
 <html>
   <head>
+
+    <div>
+      {{!-- our logo and title --}}
+      <nav>
+      <form action="/movies/search" method="POST">
+    <input type="text" name="search" placeholder="Search you Movie here!">
+    <button type="submit">Search</button>
+</form>
+      <ul>
+        <li><a href="/"> Home</a></li>
+        <li>{{#if user}}
+          <h3>Hello {{ user.name }}</h3>
+             <a href="auth/logout">Logout</a></li>
+        <li> <a href=""> My Profile</a></li>
+        <li>{{else}}
+   <a href="auth/register">Sign up</a></li>
```

Comment:
the href link, needs to begin with "/auth/register"
that it's why it's always adding a new auth to the url

this need to be updated in logout and register also